### DISCUSSION TITLE
Use with a Raymarine Axiom

### DISCUSSION THREAD

**Discussion by L-Walker** [peterantypas/maiana#120] [Q&A]
I have a 5m RHIB and have wanted to add AIS to it for years, but the GME or Raymarine ones are a practically robbery for what they are. Came across this product and god it was a breath of fresh air. Take my money!

Wanted to ask two questions though:
1. Will the unit output the ais contacts to the NMEA 2000 network if I buy the NEMA 2000 base station? If so that’s perfect because my new Raymarine MFDs can display the AIS contacts really nicely over its charts. Just wanted to check it wasn’t a Tx only type setup.
2. How does it accept route data to transmit? From experience if I have a voyage running in the chart-plotter, the AIS transceiver picks that up off the NEMA 2000 and transmits it via AIS. Is this a Class A only thing that this unit wont support? Or am I worrying about nothing?

Awesome work Pete.

**Comment by peterantypas** [comment]
Yes on 1, no on 2. Class B devices do not transmit route data. It's a class A thing.
